stercket-db.sql

```diff
@@ -0,0 +1,14 @@
+--
+-- Structure de la table `stercket`
+--
+
+CREATE TABLE `stercket` (
+    `id` INT UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(100) NOT NULL,
+    `specie` VARCHAR(100) NOT NULL,
+    `type` VARCHAR(100) NOT NULL,
```

Comment:
The specie, type and owner fields are probably some kind of enumerations (types that have a known finite set of possible values)
VARCHAR is probably not the best way to handle this.

You could use the mysql ENUM type (but it's not standard SQL, it's mysql so not so compatible)
see : https://dev.mysql.com/doc/refman/8.0/en/enum.html

Or you could store integer in the Database and have constants in your code to make them readable.